awx_collection/plugins/modules/tower_user.py

```diff
@@ -8,7 +8,7 @@
 __metaclass__ = type
 
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
+ANSIBLE_METADATA = {'metadata_version': '1.2',
```

Comment:
I don't think this is the right way to do this. The version_added should reference the collection version and not the metadata version too.

Reply:
Okay, fixed, thanks.